example/components/webgl/webgl.tsx

```diff
@@ -35,35 +35,50 @@ import actions from "../../redux/actions";
 import store from "../../redux/store";
 
 import { WebGLProps } from "./webgl.types";
+import { Scene, Vector3 } from "three";
 
 const WebGL: FC<WebGLProps> = ({ three, disabled }) => {
   const threeRootElement = useRef<HTMLCanvasElement | null>(null);
   const router = useRouter();
 
+  const hideCheckmarks = (scene: Scene): void => {
+    const keys = ["Checkmark", "Light", "Shadow"];
+    const targets = getChildren(scene, keys, "partial");
```

Comment:
Eens kijken als je hier de parent van een checkmark kan selecteren (parent in Blender). Dan heb je een mesh/group waarvan je zijn visibility of de visibility van de children van kan aanpassen